src/content/site/bots.md

```diff
@@ -157,12 +187,71 @@ N/A
 `re!valid` - Tests whether the given regex is valid.
 
 
+
 ## StandupBot
 
 **Description:** StandupBot is a bot that controls the standup channel. It enforces a strict message format. Once you have successfully sent a message you will no longer be able to post in the standup channel for 20 hours.
 
+**Source:** https://github.com/the-programmers-hangout/standup
+
 **Prefix:** N/A
 
 ### Commands
 
 N/A
+
+
+
+## Passive bots
+
+Below are the bots you will see in the server but you will not need to interact with them. They automatically do their job!
+
+## EmbedBot
+
+**Description:** EmbedBot allows staff to manage and post embed messages into various channels.
+
+**Source:** https://github.com/JakeJMattson/EmbedBot
+
+
+## Emourge
+
+**Description:** Emourge tracks emote usage. We use this to help us identify the least used emotes incase we want to add some new ones!
+
+**Source:** https://github.com/dfireBird/emourge
+
+
+## Hawk
+
+**Description:** Hawk manages nickname prefixes & suffixes. It adds 🔨 to staff nicks and organises the occasional nick party!
+
+**Source:** https://github.com/the-programmers-hangout/Hawk
+
+
+
+## Nano
+
+**Description:** Nano manages our AMAs. Whenever someone is accepted for an AMA, we'll give them a run down of how it works.
+
+**Source:** https://github.com/the-programmers-hangout/Nano
+
+
+## Skadi
+
+**Description:** Skadi manages activity roles on the server.
+
+**Source:** This source isn't publically available. We don't want people trying to game the system.
+
+
+## Taboo
+
+**Description:** Taboo makes sure only accepted file formats are uploaded.
+
+**Source:** https://github.com/the-programmers-hangout/Taboo
+
+
+
+## Watchdog
+
+**Description:** Watchdog guards the gates of our server against raids and bots.
+
+**Source:** This source isn't publically available.
```

Comment:
publicly

> The Merriam-Webster’s usage guide concludes its entry on “publically” with this advice: “You can use it if you like, but we do not really recommend it, because it will look unfamiliar to many who encounter it.”

Reply:
Fixed